gridcapa-task-manager-app/src/main/resources/application.yml

```diff
@@ -25,6 +25,8 @@ spring:
           destination: task-manager-logs-exchange
         task-updated:
           destination: task-updated
+        task-status-updated:
+          destination: task-updated
```

Comment:
avoid using a destination value already used (could lead developers using the server into error), what about task-dto-status-updated